### ISSUE TITLE
Small offset in every paragraph

### ISSUE THREAD

**Issue by liady** [liady/ChatGPT-pdf#2]
There is a small offset in the top of every paragraph when exporting. Probably an issue with `html2canvas` as well

**Comment by adrianmarinwork** [comment]
Hey, I would like to take care of this issue.

**Comment by liady** [comment]
@adrianmarinwork sounds good! Feel free to open a PR :)

**Comment by adrianmarinwork** [comment]
There is something happening, when running `git clone` of the fork I created from your repository I get this error and no files are coming from the repo.

![image](https://user-images.githubusercontent.com/118568289/205652620-159718d0-60a5-496a-9cb1-b1e6ceff9739.png)

**Comment by liady** [comment]
@adrianmarinwork can you try now?

**Comment by adrianmarinwork** [comment]
Everything works fine now, thank you!

**Comment by adrianmarinwork** [comment]
One question, do I have a way to run the project locally or something to test?

**Comment by liady** [comment]
Yes, of course, you can open `chrome://extensions` and load the extension from the src folder where you cloned the repo. That way you can change the code, and then just reload the extension and you'll have it updated.
<img width="446" alt="image" src="https://user-images.githubusercontent.com/7003853/205666895-2e5fe13e-7d3c-4e46-857b-fca33637a803.png">

**Comment by adrianmarinwork** [comment]
Hello, hello.

Looks like I've managed to make a fix for this issue of html2canvas library.

What do you think? 

![image](https://user-images.githubusercontent.com/118568289/205930002-27b1b373-ab40-466a-ad25-17649bcb67ef.png)

**Comment by liady** [comment]
Looks good! Though the prompts are a bit too high now, aren't they?
How did you manage to fix it?

**Comment by adrianmarinwork** [comment]
If you want I can explain to you how I've fixed this, do you have another place where we can message faster and easier?

**Comment by liady** [comment]
![image](https://user-images.githubusercontent.com/7003853/205935488-0d2f98c8-689a-434f-8df2-8d4a369bb294.png)
Looks like the "questions" now sit a bit higher than the "answers"

**Comment by liady** [comment]
@adrianmarinwork you can open a PR for this fix, of course :)

**Comment by adrianmarinwork** [comment]
Yeah, you are right, but I think that this comes from the own ChatGTP, if you pay attention I think that happens the same with the normal chat.

![image](https://user-images.githubusercontent.com/118568289/205939278-8b4a20f2-c0db-4945-afc7-94ea09fc478e.png)

**Comment by liady** [comment]
@adrianmarinwork yes, you're right. Thank you for the fix! Do you want to merge it to the extension?

**Comment by adrianmarinwork** [comment]
I've created a pull request with the bible explaining you everything!

**Comment by adrianmarinwork** [comment]
Wait before merge pls, I've found an issue

**Comment by adrianmarinwork** [comment]
Done, issue fixed! Feel free to merge.

**Comment by liady** [comment]
Looks good! I suggested converting it to forEach, let me know what you think

**Comment by adrianmarinwork** [comment]
Yeah, why not.

**Comment by liady** [comment]
Fixed by #7 (@adrianmarinwork )

**Comment by adrianmarinwork** [comment]
By the way, your projects looks pretty cool, I like the idea and the job that it does! If you need any more help or you are searching for someone to work with don't doubt in calling me!

**Comment by liady** [comment]
@adrianmarinwork , any idea why this still happens?
![image](https://user-images.githubusercontent.com/7003853/205988723-b65272c6-f361-4506-9ccd-908cce679d6c.png)

**Comment by adrianmarinwork** [comment]
Ouuu, it's probably because the container that contains the text it's different from the normal text, check the getElementeByClass, I'll take a look when I'm back home!

**Comment by adrianmarinwork** [comment]
No way @liady, looks like there was an easier way of fixing this problem maaaan.
The first time that I found [this fix](https://github.com/niklasvh/html2canvas/issues/2775) I tried it but for some reason, it didn't work out, I think it was because I got confused or I reloaded the page by mistake, but if we add this style property `line-height: 0.5` to the body of the page all the text stays in his place when generating the image.
I also recall trying it in the container `elements.thread`, but looks like it only works when you do it in the `body`. 
Looks like even if we add it in the main body of the page, nothing from the original page breaks.

![image](https://user-images.githubusercontent.com/118568289/206538778-c577fdf6-2666-4595-a9a1-6aa11903aa02.png)
